robottelo/config/validators.py

```diff
@@ -225,7 +225,11 @@
         Validator(
             'oscap.content_path',
             must_exist=True,
-        )
+        ),
+        Validator(
+            'oscap.profile',
+            default='security7',
```

Comment:
Add `must_exist` validation as anyone can leave it blank in config.

Reply:
Isn't that why we have the default value?

Reply:
Yup, it does but for good practice its good to keep the value in conf file as one can understood whats in conf file instead of looking at this constant file.

Reply:
alright, done.